frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/storage/DisksAllocationModel.java

```diff
@@ -366,8 +367,9 @@ private void updateImageToDestinationDomainMap() {
             } else if (diskModel.getVolumeType().getIsAvailable()) {
                 VolumeType volumeType = diskModel.getVolumeType().getSelectedItem();
                 diskImage.setVolumeType(volumeType);
-                diskImage.setVolumeFormat(AsyncDataProvider.getInstance().getDiskVolumeFormat(
-                        volumeType, storageDomain.getStorageType()));
+                diskImage.setVolumeFormat(diskImage.getBackup() == DiskBackup.Incremental ? VolumeFormat.COW :
```

Comment:
it would be better to put that check in AsyncDataProvider#getDiskVolumeFormat so it will apply to all other places, but we can do that separately